plugins/package-managers/node/src/main/kotlin/NodePackageManager.kt

```diff
@@ -34,9 +34,9 @@ import org.ossreviewtoolkit.model.utils.DependencyGraphBuilder
 import org.ossreviewtoolkit.utils.common.realFile
 
 abstract class NodePackageManager(val managerType: NodePackageManagerType) : PackageManager(managerType.projectType) {
-    protected abstract val graphBuilder: DependencyGraphBuilder<*>
+    internal abstract val graphBuilder: DependencyGraphBuilder<*>
```

Comment:
I believe the commit-scope should be "node" for this.

Also, out of curiosity, why is the commit-type "refactor" here, but "chore" for the next commit? Shouldn't both be of the same type, as the changes are very similar in type?

Reply:
Hmm, I'm confused about the commit message, which says to "reduce the visibility". Because isn't `internal` actually *increasing* the visibility compared to `protected`? For [class members](https://kotlinlang.org/docs/visibility-modifiers.html#class-members), "`protected` means that the member has the same visibility as one marked as `private`, but that it is also visible in subclasses" and "`internal` means that any client *inside this module* who sees the declaring class sees its `internal` members".

Reply:
> Hmm, I'm confused 

The rationale applies, because if I use an `internal` type as generic type parameter for the graph builder, then I get a compile error. It complains, that the `internal` property is exposed outside of the module.

So, from that it seems that internal is more limiting.

Reply:
Unfortunately, your answer even confuses me more:

> if I use an `internal` type as generic type parameter for the graph builder, then I get a compile error.

But you *are* now using `internal` for the `graphBuilder`, and obviously it compiles.

> So, from that it seems that internal is more limiting.

I guess you're saying: `protected` would be visible also to subclasses *outside* of the module, but `internal` limits visibility to the module. Is it that what you're after?

Reply:
> I guess you're saying: `protected` would be visible also to subclasses _outside_ of the module, but `internal` limits visibility to the module. Is it that what you're after?

Yes, exactly that. (I made a "typo" in my previous comment.)

Reply:
@sschuberth : your commit [1] made the module infos public, most likely because of that mentioned compile error.

[1]: https://github.com/oss-review-toolkit/ort/commit/7b21289f922c470be794e443cb104b042733ccc5#diff-25049a7a0b1e62fd30436163728124840da28f2b7dd18b751fac122deb8a221fR34